modules/change-collection-method-ebpf.adoc

```diff
@@ -3,17 +3,17 @@
 // * upgrading/upgrade-operator.adoc
 :_mod-docs-content-type: CONCEPT
 [id="change-collection-method-ebpf_{context}"]
-= Changing the collection method to EBPF
+= Changing the collection method to CORE_BPF
 
 [role="_abstract"]
-If the cluster that you are upgrading contains the `SecuredCluster` CR, you must ensure that the per node collection setting is set to EBPF before you upgrade.
+If the cluster that you are upgrading contains the `SecuredCluster` CR, you must ensure that the per node collection setting is set to CORE_BPF before you upgrade.
 
 .Procedure
 
 . In the {ocp} web console, go to the {product-title-short} Operator page.
 . In the top navigation menu, select *Secured Cluster*.
 . Click the instance name, for example, *stackrox-secured-cluster-services*.
-. Click YAML to open the YAML editor.
-. Locate the `spec.perNode.collector.collection` attribute.
-. If the value is `KernelModule`, then change it to `CORE_BPF`.
-. Click *Save*.
\ No newline at end of file
+. Use one of the following methods:
+* In the *Form view*, under *Per Node Settings* -> *Collector Settings* -> *Collection*, select *CORE_BPF*.
+* Click *YAML* to open the YAML editor and locate the `spec.perNode.collector.collection` attribute. If the value is `KernelModule`, then change it to `CORE_BPF`.
```

Comment:
The comments here are similar to those of 4.3. The difference here is that the collection option doesn't actually do anything, unless the forceCollection option is used. All customers are switched to CORE_BPF unless they select the forceCollection option and EBPF. However, it is still good if the collection method in the yaml and form matches what is actually used in the cluster. This information should be be conveyed here, though I think it is already included elsewhere.